bgpd/bgp_nht.c

```diff
@@ -1103,6 +1103,11 @@ void evaluate_paths(struct bgp_nexthop_cache *bnc)
 					bgp_evpn_import_route(bgp_path,
 						afi, safi, bgp_dest_get_prefix(dest), path);
 			}
+		} else if (safi == SAFI_EVPN &&
+			   bgp_evpn_is_prefix_nht_supported(
```

Comment:
So if path_valid is false ( no nexthops are workable ) and bnc_is_valid_nexthop is false and if the route is evpn, we import it?  This seems broken.  How is this correct?

Reply:
Fixed

Reply:
not the same code